core/src/main/java/org/elasticsearch/transport/ConnectionProfile.java

```diff
@@ -42,14 +44,16 @@
             TransportRequestOptions.Type.PING,
             TransportRequestOptions.Type.RECOVERY,
             TransportRequestOptions.Type.REG,
-            TransportRequestOptions.Type.STATE)), 1);
+            TransportRequestOptions.Type.STATE)), 1, null);
```

Comment:
Could we document where/when this light profile is used? For what I see it's used to connect to nodes on zen pings & transport client sniffer. Also, maybe we could use a lower timeout?